DigitalRain/Raindrops/IRaindropFactory.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace DigitalRain.Raindrops
+{
+    using Columns;
+
+    public interface IRaindropFactory
```

Comment:
Thank you for making this an interface. If we are going to have 1 million factories we should make sure the interface is well defined for my smooth brain to comprehend it all

Reply:
Yep, more interfaces will emerge over time.  First there's just one implementation for a thing, but as soon as there's two, bam! Interface